dev-support/Jenkinsfile_GitHub

```diff
@@ -144,8 +143,9 @@ pipeline {
                         YETUS_ARGS+=("--docker")
                         YETUS_ARGS+=("--dockerfile=${DOCKERFILE}")
                         YETUS_ARGS+=("--mvn-custom-repos")
-                        YETUS_ARGS+=("--multijdkdirs=/usr/lib/jvm/java-8-openjdk-amd64")
-                        YETUS_ARGS+=("--findbugs-home=/usr")
+                        YETUS_ARGS+=("--java-home=/usr/lib/jvm/java-8")
+                        YETUS_ARGS+=("--multijdkdirs=/usr/lib/jvm/java-8,/usr/lib/jvm/java-11")
+                        YETUS_ARGS+=("--multijdktests=compile,unit")
```

Comment:
huh. I thought multijdk handling of unit tests didn't work. ref: https://s.apache.org/fqjcn

Reply:
I'm not sure what to tell you. This looks like explicit support to me, at least for running the unit tests with all supplied JDKs.

https://github.com/apache/yetus/blob/138de2232a351e9d9f4c3d751e7d25e52804dfe0/precommit/src/main/shell/test-patch.sh#L2103-L2105

Reply:
Hurm. The test report still looks like a mixed bag. The test failures table shows both jdks, but the test vote table only has jdk8. The footer entry with the link to unit test console output also only has jdk8.

Reply:
Okay. Let me break the JDK11 check out of a multijdk block and into separate stages. Do you prefer parallel or serial stages?

Reply:
I personally prefer parallel. especially once we get the github integration in place to list the tests and check them off as we go it'll be nice.

however, if we do serial I believe jenkins will optimize executor reuse. so that might give a better experience (both us being nice to others and minimizing total time to be "done") prior to that incremental progress reporting.

do whichever you like.

Reply:
> Hurm. The test report still looks like a mixed bag. The test failures table shows both jdks, but the test vote table only has jdk8. The footer entry with the link to unit test console output also only has jdk8.

You're right. Have a look at the table printed over on https://github.com/apache/hbase/pull/1214#issuecomment-591757259. This should be filed as a bug/feature in Yetus?

> do whichever you like.

I've started plumbing in for parallel steps, since that's what nightly does and I can follow along. This is more involved than I originally imagined. I'm not sure how reporting of each check's success will work out -- right now we have a single summary written by Yetus, after this I expect we'll have 3 summaries (general, jdk8, jdk11), each written back independently. I'm inclined to not bother also collecting and writing a summary and the reports, given that we want to invest in HBASE-23902.